### DISCUSSION TITLE
Share a link to your JBrowse 2 instance

### DISCUSSION THREAD

**Discussion by cmdcolin** [GMOD/jbrowse-components#3917] [Show and tell]
Share a link (or screenshot!) of your JBrowse 2 instance, and tell us:

- what species or data types you are using
- what features of JBrowse 2 you or your users like the most
- interesting config customizations or plugin usage
- anything else you'd like to share!

**Comment by tberardini** [comment]
jbrowse2.arabidopsis.org

Arabidopsis thaliana

still have some tracks that show up in JBrowse but not JBrowse2
announcing to our community that we're retiring SeqViewer has brought up some feature requests for JBrowse2
https://github.com/GMOD/jbrowse-components/discussions/3346
https://github.com/GMOD/jbrowse-components/issues/3873
https://github.com/GMOD/jbrowse-components/issues/3872

**Comment by kimrutherford** [comment]
Thanks very much for JBrowse V2.  It's great!  We have started configuring it for *S. pombe*: https://www.pombase.org/jbrowse2/

It's still in progress so we haven't announced it to our users yet.  

Thanks for making things easy by providing jbrowse-cli.  It's very helpful to run `jbrowse add-track` to get an example configuration for each of the file types we have (BAM, VCF, BED, GFF3, bigwig).

We have some Hi-C datasets that we're looking forward to trying later.

**Comment by jhumann** [comment]
I just got us converted on the Genome Database for Vaccinium.  Working on formatting pangenome data to add to 3 of the genomes.
https://www.vaccinium.org/jbrowse2/

**Comment by burtsev** [comment]
Thanks a lot for JBrowse! GUI of our platform is build around it and we are looking forward for any feed back how we can improve data representation with JBrowse for various tasks we have. 

The link is here - https://genomicintelligence.ai/ 
<img width="1930" height="925" alt="image" src="https://github.com/user-attachments/assets/8a769f9c-4007-4af9-ab75-e65d100f0566" />

**Reply by cmdcolin** [reply]
looks awesome! I got "NetworkError when attempting to fetch resource." when trying to click the gene search buttons like TP53 maybe from cors error from ensembl
